cartridge/graphql.lua

```diff
@@ -346,6 +367,42 @@ local function init(httpd)
     )
 end
 
+--- Set up trigger for GraphQL handlers.
+--
+-- It will be executed **before** top-level resolvers, which were
+-- registered using `add_callback` or `add_mutation` methods.
+--
+-- The trigger function is called with one argument:
```

Comment:
Why only one?

I suggest you pass it as separate arguments (without any temporary tables) and pass http request object as well